frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.styled.tsx

```diff
@@ -165,16 +164,16 @@ export const Footer = styled.footer<{ variant: FooterVariant }>`
   align-items: center;
   ${props => footerVariantStyles[props.variant]}
   height: calc(50 / 16 * 1rem);
-  padding: 0 0.5rem;
+  padding: 0 1em;
 
-  ${breakpointMinMedium} {
+  ${breakpointMinSmall} {
```

Comment:
I based the breakpoints on the header.

https://github.com/metabase/metabase/blob/b54bc40cfd54ab24a693efc117fe601bed7c8d8b/frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.styled.tsx#L53-L63

it uses small and large. So I make the footer and the margin in the file below use the same breakpoints because I don't know why they're all different.